statistics/shortest_time_to_get_all_singles.rb

```diff
@@ -3,7 +3,8 @@
 
 class ShortestTimeToGetAllSingles < Statistic
   def initialize
-    @title = "Shortest Time to Get All Singles"
+    @title = "Shortest time to get all singles"
+    @note = "Only current official events are taken into account."
```

Comment:
I think it may be confusing to some people that even though they completed all official events in the past (e.g. before skewb was introduced) they don't appear on the top. In such cases it's worth adding a clarification.